kr-core/src/main/clojure/edu/ucdenver/ccp/kr/sparql.clj

```diff
@@ -243,11 +244,43 @@
        (sparql-query-body optional-clauses)
        " }\n "))
 
+(defn filter-body [filter-clauses]
+  (str " FILTER ( "
+       (operator-body filter-clauses)
+       " )\n "))
+
+(defn bind-body [bind-clauses]
+  (str " BIND "
+       (operator-body bind-clauses)
+       " \n "))
+
+(defn iri-body [iri-clauses]
+  (str " IRI "
+       (operator-body iri-clauses)
+       " \n "))
+
+(defn concat-body [concat-clauses]
+  (str " CONCAT "
+       (operator-body concat-clauses)
+       " \n "))
+
+(defn strafter-body [strafter-clauses]
```

Comment:
Your summary indicates that `strafter` is buggy; can you add more detail about what's missing?